### ISSUE TITLE
oval macrocytes

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4873]
**Preferred term label:**
oval macrocytes

**Synonyms**
macroovalocytes, macro-ovalocytes, oval macrocystosis

**Definition (free text, please give PubMed ID)**
Large erythroblasts, oval shaped containing a characteristic immature, lacy nucleus.

PMID: 19202968
PMID: 28821482
PMID: 16988104

**Parent term (use hpo.jax.org/app)**

Abnormal erythrocyte morphology
HP:0001877

http://purl.obolibrary.org/obo/HP_0001877

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM: 612542
B12 deficiency 

Pancytopenia.

OMIM: 603529
DYSERYTHROPOIESIS, CONGENITAL

**Your nano-attribution (ORCID)**
0000-0001-9969-8610

**Comment by pnrobinson** [comment]
I think the parent term should be https://hpo.jax.org/app/browse/term/HP:0004447
